src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ThumbTests/Tumb_CapturePreventScroll_Tests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Linq;
+using FluentAssertions;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Controls.ThumbTests;
+
+[TestFixture]
+public partial class Tumb_CapturePreventScroll_Tests : SampleControlUITestBase
+{
+	private const string _sample = "UITests.Windows_UI_Xaml_Controls.ThumbTests.Thumb_CapturePreventScroll";
+
+	[Test]
+	[AutoRetry]
+	[ActivePlatforms(Platform.iOS, Platform.Android)]
+	public void When_Drag_Then_DoesNotScroll()
```

Comment:
Let's start using `UnoWorkItem` attribute more

Reply:
Well it's not a work item, it's only because the issue is present only for native touch targets (i.e. iOS and Android).